src/main/java/org/exist/xquery/modules/jfreechart/JFreeChartFactory.java

```diff
@@ -145,13 +145,13 @@ public static JFreeChart createJFreeChart(final String chartType, final Configur
                 setCategoryChartParameters(chart, conf);
                 break;
 
-            case "BarChart3D":
-                chart = ChartFactory.createBarChart3D(
-                        conf.getTitle(), conf.getCategoryAxisLabel(), conf.getValueAxisLabel(), categoryDataset,
-                        conf.getOrientation(), conf.isGenerateLegend(), conf.isGenerateTooltips(), conf.isGenerateUrls());
-
-                setCategoryChartParameters(chart, conf);
-                break;
+//            case "BarChart3D":
```

Comment:
Are these various chart types meant to be commented out?

Reply:
yes, as they are removed from v1.5 ; the non 3d ones look much better now (according to some reports)

Reply:
Okay - maybe just remove that code then!